query-engine/connector-test-kit/src/test/scala/util/TestServer.scala

```diff
@@ -19,7 +19,7 @@ case class TestServer() extends PlayJsonExtensions with LogSupport {
       project: Project,
       dataContains: String = "",
       legacy: Boolean = true,
-      batchSize: Int = 5000,
+      batchSize: Int = 500,
```

Comment:
TODO
Introduces uncertainty about previous value I guess?

Reply:
We don't need that, I just disabled the VeryMany tests anyhow...